docs/tr/contributing.md

```diff
@@ -0,0 +1,144 @@
+     * [Kodlama Stili](#Kodlama-Stili) hakkında aşağıya bakın.
+	
+3. 🚦 **Kodunuzu** test edin.
+     * Hataları giderirken veya özellikler eklerken gerektiği gibi birim testleri ekleyin.
+     * Test paketini ilgili paket klasöründeki `vendor/bin/phpunit` ile çalıştırın.
+	 
+<!--
+    * See [here](link-to-core/tests/README.md) for more information about testing in Flarum.
+-->
+4. 💾 Kodunuzu açıklayıcı bir mesajla **işleyin**.
+     * Değişikliğiniz mevcut bir sorunu çözüyorsa (genellikle bu, 123 numaralı sorun numarası olmak üzere yeni satırda "123 numaralı düzeltmeleri" içermelidir).
+     * Bir [iyi işlem mesajı](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) yazın.
+
+5. 🎁 GitHub'da bir Çekme İsteği (PR) **gönderin**.
+     * Çekme talebi şablonunu doldurun.
+     * Değişikliğiniz görselse, değişikliği gösteren bir ekran görüntüsü veya GIF ekleyin.
+     * JavaScript `dist` dosyalarını DERLEMEYİN. Bunlar birleştirme sırasında otomatik olarak derlenecektir.
+
+6. 🤝 Onay için Flarum ekibiyle **iletişim kurun**.
+     * Ekip üyeleri kodunuzu inceleyecek. Bazı değişiklikler veya iyileştirmeler veya alternatifler önerebiliriz, ancak küçük değişiklikler için çekme talebinizin hızla kabul edilmesi gerekir.
+     * Geri bildirimi ele alırken, üzerine yazmak veya ezmek yerine ek taahhütleri itin (birleştireceğiz).
+
+7. 🕺 **Dans et** tıpkı Flarum'a katkıda bulunduğun gibi.
+
+## Kodlama Stili
+
+Flarum kod tabanını temiz ve tutarlı tutmak için, takip ettiğimiz bir dizi kodlama stili yönergemiz var. Şüpheye düştüğünüzde kaynak kodunu okuyun.
+
+Kod stiliniz mükemmel değilse endişelenmeyin! StyleCI, herhangi bir stil düzeltmesini, çekme istekleri birleştirildikten sonra otomatik olarak Flarum depolarında birleştirir. Bu, kod stiline değil katkının içeriğine odaklanmamızı sağlar.
+
+### PHP
+
+Flarum, [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md) kodlama standardını ve [PSR- 4](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-4-autoloader.md) otomatik yükleme standardı. Bunun da ötesinde, [diğer stil kurallarına](https://github.com/flarum/core/blob/master/.styleci.yml) uyarız. Mümkün olduğunda PHP 7 tür ipucu ve dönüş türü bildirimlerini ve satır içi belgeler sağlamak için [PHPDoc](https://docs.phpdoc.org/) kullanıyoruz. Katkılarınızda kod tabanının geri kalanı tarafından kullanılan stili deneyin ve taklit edin.
+
+* Ad alanları tekil olmalıdır (ör. `Flarum\Discussion`, not `Flarum\Discussions`)
+* Arayüzlerin sonuna `Interface` eklenmelidir (ör. `MailableInterface`)
+* Abstract sınıflarının önüne `Abstract` yazılmalıdır (ör `AbstractModel`)
+* Özelliklerin sonuna `Trait` eklenmelidir (ör. `ScopeVisibilityTrait`)
+
+### JavaScript
+
+Flarum'un JavaScript'i çoğunlukla [Airbnb Stil Kılavuzu](https://github.com/airbnb/javascript)'nu takip eder. Satır içi dokümantasyon sağlamak için [ESDoc](https://esdoc.org/manual/tags.html) kullanıyoruz.
```

Comment:
"Doküman" yerine "belge" diyebiliz.

Reply:
Thanks, fixed it.

- https://github.com/flarum/docs/pull/154/commits/bb0c788c46394d8acac0a45c1172ffc627dfd33e